client/my-sites/site-settings/jetpack-credentials/credentials-setup-flow/setup-footer.jsx

```diff
@@ -51,4 +62,8 @@ class SetupFooter extends Component {
 	}
 }
 
-export default localize( SetupFooter );
+export default connect( null, {
+	happychatEvent: withAnalytics(
+		recordTracksEvent( 'rewind_credentials_get_help', {} )
+	),
```

Comment:
In this case we won't need `withAnalytics()` because we aren't dispatching any other Redux actions. Actually, this won't work because `withAnalytics()` is expecting a second arg.

For now, before we have any additional Redux actions, we can just pass `recordTracksEvent()` inside of `connect()`